tests/network/wireguard.pm

```diff
@@ -37,7 +37,10 @@ sub run {
         $remote     = '10.0.2.101';
     }
 
-    zypper_call 'in wireguard-kmp-default wireguard-tools';
+    assert_script_run 'grep -i wireg /boot/config-$(uname -r)';
```

Comment:
Nit: I put it like this in the chat, but maybe for the sources it should be:
`grep CONFIG_WIREGUARD /boot/config-$(uname -r)`

Reply:
True, it looks better and is probably more understandable.

Reply:
nit: \+ now it can be without `-i` :).